src/DotLiquid/Util/R.cs

```diff
@@ -79,7 +92,7 @@ public static List<string> Scan(string input, string pattern)
         /// <param name="input"></param>
         /// <param name="pattern"></param>
         /// <param name="callback"></param>
-        /// <returns></returns>
+        [Obsolete("Use Scan(string, Regex, Action) instead.")]

```

Comment:
Should we add a call to this in RegexpTests to avoid the code coverage drop?

Reply:
Ah - I didn't see any tests for the old function, so it didn't register that the coverage would drop, but we're testing it indirectly... I'll add a test.